doc/helpers/VisualState-extensions.md

```diff
@@ -0,0 +1,110 @@
+# VisualStateExtensions Attached Properties
+Exposes visual states as attachable properties that lets you bind a string on a `Control` to set its `VisualState`s.
+
+## Remarks
+`VisualStateManager.GoToState` is typically used with `Control` where you would set `<VisualStateManager.VisualStateGroups>` on the root element of the ControlTemplate. Because this class is implemented using the same method, it means that if you are setting `Group1State` on an element, the `VisualStateManager.VisualStateGroups` should not be set on the very same element, but its first child:
+```xml
+<Page utu:VisualStateExtensions.Group1State="{Binding OnboardingState}">
+    <!-- Wrong -->
+    <VisualStateManager.VisualStateGroups>...
+
+    <!-- Good -->
+    <Grid>
+        <VisualStateManager.VisualStateGroups>...
+```
+
+Multiple `Group1..4State`s are provided:
+1. to allow for multiple concurrent/parallel states (eg: LoginStates and NetworkStates groups) where each group affect different sets of UI elements' proties; and
```

Comment:
```suggestion
1. to allow for multiple concurrent/parallel states (eg: LoginStates and NetworkStates groups) where each group affects a different set of UI elements' properties; and
```

Reply:
Should it be `UIElement` instead of "UI element" or you just mean properties relating to specific components of the UI?

Reply:
this reads like I had a stroke lol

Reply:
In the board sense, not the `UIElement`.